src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :first-child {
+  background-image: url(images/star-active.svg);
+  
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+

```

Comment:
![image](https://user-images.githubusercontent.com/99331793/200772109-292c1026-b590-405c-8d30-0198c6da4ea9.png)

Reply:
changed it